assembly/bindgen.ts

```diff
@@ -374,12 +381,13 @@ function decode<T, V = Uint8Array>(buf: V, name: string = ""): T {
         " of type map expected a JSON.Obj, but recevied " +
         JSONTypeToString(val)
     );
-    //@ts-ignore
     assert(
+      //@ts-ignore
       nameof<indexof<T>>() == "String",
       "Value with Key: " +
         name +
         " cannot decode a map which has an index type " +
+        //@ts-ignore
```

Comment:
everywhere else here, too 🙂 

strange that all of these needed to be moved/added – I think specifying specific rules to ignore might make that more understandable when reviewing PRs like this

Reply:
Yeah I wish too.  And truly look forward to better IDE support.  Do you think it would be weird to move them to the end of the line so that they don't get in the way of reading?

Reply:
I think that would be great!

Reply:
So I just tried and it looks like they go back after auto-lint fixes are applied. So we can either ignore this file for linting or just forget about it for now.